#test exception 8

int main() {
    var x, y : int;
    x = 0;
    y = 10;
    if(y % x > 0 || 1) // make sure it doesn't short circuit the wrong way
        return 1;
    return 0;
}